ui/README.md

```diff
@@ -1,47 +1,82 @@
 # Perses UI
 
 This serves as the monorepo root for the packages that make up the Perses UI.
-Currently, those packages consist of:
+
+## UI quick start
+
+If you are new to Perses, please review the rest of this README first before starting development.
+
+### Perses app
+
+- Start the backend server in development mode by running this script from the project root: `./scripts/api_backend_dev.sh`
+- Change to the `ui` directory.
+- Install dependencies: `npm install`
+- Start the Perses app: `npm run start`
+- Open the app: http://localhost:3000/
+
+### Storybook
+
+- Change to the `ui` directory.
+- Install dependencies: `npm install`
+- Start the storybook: `npm run storybook`
+- Open the storybook: http://localhost:6006/
+
+### Perses app and storybook
+
+Follow the steps for "Perses app", but start using this script: `npm run start-with-storybook`.
+
+## Package organization
+
+The UI-based code for Perses is organized as a monorepo using [turborepo](https://turbo.build/repo).
+
+The root `package.json` in `ui` has
+[NPM workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces) enabled, so running `npm install` will install
+dependencies for all packages and add symlinks to the local versions for packages that depend on other packages in the
+workspace. The root `package.json` also contains `devDependencies` that are commonly used across multiple packages in
+the workspace.
+
+Perses is broken up in to a number of separate packages to allow for flexibility when embedding functionality. You can generate a graph of the dependencies by using the [turbo cli's `--graph` option](https://turbo.build/repo/docs/reference/command-line-reference#--graph). For example, `npm run start -- --graph=graph.svg` will output a graph of the dependencies of the `start` command in SVG format.
+
```

Comment:
Organizing these to clarify the three types of packages in this project because we have different expectations for each of them and it makes it easier to reference them by category.